ert_shared/storage/blob_api.py

```diff
@@ -45,3 +45,14 @@ def add_blob(self, data):
 
     def get_blob(self, id):
         return self._session.query(ErtBlob).get(id)
+
+    def get_blobs(self, ids):
```

Comment:
How should this be handled if some ids are not present in the database?

Reply:
At this level I think it is reasonable to expect to get an empty result if one asks for something that doesn't exist. The public http api, as it is after the latest changes and if those changes remain standing, does not make it possible to ask for this directly by a user.